cmd/fleetctl/user.go

```diff
@@ -174,6 +174,10 @@ func createUserCommand() *cli.Command {
 				return fmt.Errorf("Failed to create user: %w", err)
 			}
 
+			if apiOnly && sessionKey != nil && *sessionKey != "" {
+				fmt.Printf("Success! The API token for your new user is: %s\n", *sessionKey)
```

Comment:
I think we want the key visible because if you're creating an API-only user you probably intend to copy/paste it elsewhere momentarily, so need an easy way to get it. (If users express concerns we might consider iterating on this by hiding the API key but automatically copying to the user's clipboard. But for now, the goal is just to make it simpler to get that API key.) 

cc @noahtalerman

Reply:
Copy can be secure as well using something like pbcopy:

```
fleetctl user create ... | pbcopy
```